.gitignore

```diff
@@ -4,6 +4,9 @@ node_modules/
 # Eleventy output directory.
 _site/
 
+# Component Metadata (generated)
+custom-elements.json
```

Comment:
This is generated ever time we run `npm run storybook` now, so it can make for some really noisy PRs if you don't exclude it.